### PR TITLE
#NEAF-93 rewrite districts dao to spark sql

### PR DESCRIPTION
**Pull request by VladimirBaklan**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -28,7 +28,7 @@ libraryDependencies ++= Seq(
   "com.typesafe.play" %% "play-json" % "2.7.3",
   "info.debatty" % "java-string-similarity" % "1.2.1",
   "org.slf4j" % "slf4j-api" % "1.7.26",
-  "org.slf4j" % "slf4j-nop" % "1.7.26" % Test
+  "org.slf4j" % "slf4j-nop" % "1.7.26" % Test,
```
Comment (sleshJdev) [inline build.sbt:31]: Is this change needed?

Comment (VladimirBaklan) [reply build.sbt:31]: I tested some dependency's and forget to remove it. If it's a problem, I will remove it

**Review by sleshJdev — APPROVED** [review]

**Comment by VladimirBaklan** [comment]
Will be implemented in NEAF 91, so, no need to merge it. Closed
